numcodecs/vlen.c

```diff
@@ -25979,7 +25979,7 @@ static const char* __Pyx_BufFmt_DescribeTypeChar(char ch, int is_complex) {
     case 'P': return "a pointer";
     case 's': case 'p': return "a string";
     case 0: return "end";
-    default: return "unparseable format string";
+    default: return "unparsable format string";
```

Comment:
Yeah, let's omit this one, esp. considering it's in the output.

Reply:
OK, removed that commit.